.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "liveServer.settings.NoBrowser": true,
-  "liveServer.settings.port": 5501
+  "liveServer.settings.port": 5502
 }
```

Comment:
- [ ] [OPTIONAL] remember you can add this file to your .gitignore file. Since this is simply a VS Code settings file it is not required to be tracked in your repository.

Reply:
okay, Thank I did that